app/controllers/contributors_controller.rb

```diff
@@ -69,10 +71,12 @@ def conversations
   private
 
   def set_contributor
+    # TODO: scope by organization?
```

Comment:
Same as :point_up: in that I'm not sure that we need to scope it, but maybe we prefer to not have these lookups of the entire db?

Reply:
It can be abused if user manipulates url. Fixed with: https://github.com/tactilenews/100eyes/pull/1987

Reply:
True, good point. This is where authorization comes in handy.

Reply:
done with: https://github.com/tactilenews/100eyes/pull/1987